core/src/main/java/bisq/core/dao/DaoSetup.java

```diff
@@ -34,22 +36,44 @@
 import bisq.core.dao.node.BsqNode;
 import bisq.core.dao.node.BsqNodeProvider;
 import bisq.core.dao.node.explorer.ExportJsonFilesService;
+import bisq.core.dao.state.DaoStateListener;
 import bisq.core.dao.state.DaoStateService;
+import bisq.core.dao.state.model.blockchain.Block;
+import bisq.core.locale.Res;
 
 import com.google.inject.Inject;
 
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.beans.value.ChangeListener;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.function.Consumer;
 
+import lombok.Getter;
+
 /**
  * High level entry point for Dao domain.
  * We initialize all main service classes here to be sure they are started.
  */
-public class DaoSetup {
+public class DaoSetup implements DaoStateListener {
```

Comment:
I think it would be better to make a separate BSQSyncService or something like that, it feels wrong to add this to the DaoSetup.

Reply:
I would prefer to not use DaoSetup for that but DaoFacade (or some custom class). I will have a look soon into it to make a more concrete sugestion.

Reply:
NACK

Reply:
Yes, I also wasn't quite happy to put it in this place, I just though to make it consistent how it is handled with the btcInfo WalletSetup right now:
https://github.com/bisq-network/bisq/blob/8cdc7599031a159b5c311904299136ba14b8a0a0/core/src/main/java/bisq/core/app/BisqSetup.java#L341-L348
https://github.com/bisq-network/bisq/blob/8cdc7599031a159b5c311904299136ba14b8a0a0/core/src/main/java/bisq/core/app/WalletAppSetup.java#L68

How do we differentiation between a Facade and a Service atm in Bisq?
I'll have a look to find a better place for this presentation logic. I might change the implementation of the btcInfo/btcSyncProgress on the way as well.

Reply:
I moved the code into the DaoPresentation class which I created already for another DAO related PR.